src/Controller/Produto/CadastrarProdutoController.php

```diff
@@ -46,6 +46,21 @@ public function salvar(Request $request): Response
         }
 
         $request = $request->request;
+        
```

Comment:
pode criar as variaveis assim, tu ajusta os outros lugares?
```suggestion
        list('nome' => $nome, 'quantidade' => $quantidade) = $request->request->all();
        
```